target/linux/ramips/image/mt7621.mk

```diff
@@ -203,6 +208,25 @@ define Device/dlink_dir-860l-b1
 endef
 TARGET_DEVICES += dlink_dir-860l-b1
 
+define Device/dlink_dir-878-a1
+  MTK_SOC := mt7621
```

Comment:
Rename to "SOC" due to 58e71e322bc0e0edd08157075e9022c15ae33ec5.

Reply:
ok Done